docs/docs/reference-react-dom.md

```diff
@@ -9,6 +9,14 @@ permalink: docs/react-dom.html
 
 The `react-dom` package provides DOM-specific methods that can be used at the top level of your app and as an escape hatch to get outside of the React model if you need to. Most of your components should not need to use this module.
 
+### Browser Support
+
+React supports most popular browsers, including Internet Explorer 9 and above.
+
+> Note
+>
+> We don't support older browsers that don't support ES5 methods, but you may find that your apps do work in older browsers if polyfills such as [es5-shim and es5-sham](https://github.com/es-shims/es5-shim) are included in the page. You're on your own if you choose to take this path.
+
  - [`render()`](#render)
```

Comment:
Maybe make this section last? Now it looks like APIs like `render()` are grouped under it.